editor/ScreenLayers/ProjectMenu.cs

```diff
@@ -241,6 +241,11 @@ public override void Load()
                 Size = new Vector2(16, 9) * 16,
             });
 
+            timeLabel.OnClick += (sender, e) => Manager.ShowPrompt("Skip to...", name => {
+                float i = 0;
```

Comment:
`var time = 0.0f;` instead.
I like var.
Don't use i j k variable names when you can give things a proper name.